api/nakadi-event-bus-api.yaml

```diff
@@ -1270,3 +1281,43 @@ definitions:
         description: Documentation for the enrichment.
     required:
       - name
+
+
+  BatchItemResponse:
+    description: |
+      A status corresponding to one individual Event's publishing attempt.
+    properties:
+      eid:
+        type: string
+        format: uuid
+        description: |
+          eid of the corresponding item.
+      publishing_status:
```

Comment:
I think the outcome of our meeting had a little bit different structure.

pushed:
 - yes
 - no

status:
 - none
 - checked
 - partitioned
 - enriched
 - published
 - failed (or maybe even several status codes for failed, showing on which step it failed)